runtime/onert/sample/minimal-python/inference_benchmark.py

```diff
@@ -45,8 +45,8 @@ def get_validated_input_tensorinfos(sess: infer.session,
     return updated_infos
 
 
-def benchmark_inference(nnpackage_path: str, backends: str, input_shapes: List[List[int]],
-                        repeat: int):
+def benchmark_inference(nnpackage_path: str, backends: str,
+                        input_shapes: Optional[List[List[int]]], repeat: int):
```

Comment:
Are the input shapes in fact optional from a particular inference perspective? Does it generate any error if Optional is not used here?